pkg/images/metrics.go

```diff
@@ -0,0 +1,33 @@
+package images
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stackrox/rox/pkg/metrics"
+)
+
+var (
+	ScanSemaphoreQueueSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "image_scan_semaphore_queue_size",
+		Help:      "A gauge vector that tracks the size of the queue for the scan semaphores used in scans.",
```

Comment:
Let me rephrase that sentence the way I understand it and you tell me whether this is correct, ok?

```suggestion
		Help:      "A gauge that shows how many scan tasks are currently waiting for the semaphore to be available.",
```

Reply:
semaphores, plural, but yes, done